src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/global.yml

```diff
@@ -57,6 +63,7 @@ Resources:
       CompatibleRuntimes:
         - python3.7
         - python3.8
+        - python3.9
```

Comment:
I am going to do an assumption here so it might be incorrect. 

The reasoning behind the PR was the ability to utilize the new advantages that 3.9 has to offer. But the shared layer stays compatible with 3.7 and 3.8. 

Therefore you cannot use the new features of 3.9 or you need build a compatible equivalent for older versions.

Is that something you have considered? Is there a specific reason for keeping the shared later compatible with other versions?

Reply:
Good point, thanks for catch that.
I will remove the older Python versions from that list.